packages/excubiae/contracts/README.md

```diff
@@ -57,23 +57,28 @@ yarn add @zk-kit/excubiae
 To build your own Excubia:
 
 1. Inherit from the [Excubia](./Excubia.sol) abstract contract that conforms to the [IExcubia](./IExcubia.sol) interface.
-2. Implement the `_check()` method to define your own gatekeeping logic and to prevent unwanted access (sybils, double checks).
+2. Implement the `_check()` and `_pass()` methods logic defining your own checks to prevent unwanted access (sybils, double checks).
```

Comment:
maybe instead of double check (which now is techincally possible by calling check directly), we should make it more clear, maybe "double pass"? or even expand a bit more and explain that ideally pass/_pass should record the data used to pass the check and prevent the same data from being used twice

Reply:
yes, that's a good point. In fact, the `check()` is not doing besides answering with "you're eligible / you're not eligible". Therefore, instead of double check is something like "avoid to pass the gate twice".